src/cupid_ai/backend/api_helpers/auth/create_new_user.py

```diff
@@ -0,0 +1,13 @@
+from api_helpers.auth.helpers import generate_unique_id
+
+
+def create_new_user(conn, email, hashed_password):
+    cursor = conn.cursor()
+    cursor.execute(
+        "INSERT INTO User (email, password) VALUES (?, ?)", (email, hashed_password)
```

Comment:
can we format these like this?
"
INSERT INTO User (email, password) 
VALUES (?, ?)
"

Reply:
even better would be to have them in a variable, and then used as cursor.execut(sql)

Reply:
Done